### PR TITLE
VR-9843 audit log interceptor move to common

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/App.java [hunk 1]
```diff
@@ -215,7 +215,7 @@ public static void main(String[] args) {
       serverBuilder.intercept(new ExceptionInterceptor());
       serverBuilder.intercept(new MonitoringInterceptor());
       serverBuilder.intercept(new AuthInterceptor());
-      serverBuilder.intercept(new AuditLogInterceptor());
+      serverBuilder.intercept(new AuditLogInterceptor(true));
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/App.java]: We don't want it always true, right? I recall some discussion around that and putting in a config.

**Review by conradoverta — APPROVED** [review]
One ask to place the flag in config. LGTM otherwise

File: backend/src/main/java/ai/verta/modeldb/App.java (hunk 1 above)
Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/App.java]: added config. Also added call count = 1 check

Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/App.java]: added change for this comment https://github.com/VertaAI/verta-uacservice/pull/561#discussion_r577395634

**Review by coryverta — APPROVED** [review]

**Review by anandjakhaniya — APPROVED** [review]

**Pull request merged**